### ISSUE TITLE
[quickstart_tutorial] why model.eval() in not turn off at training stage

### ISSUE THREAD

**Issue by guoyejun** [pytorch/tutorials#1624]
Here is the description of eval():
be sure to call model.eval() method before inferencing to set the dropout and batch normalization layers to evaluation mode. Failing to do this will yield inconsistent inference results.

In the code at https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html, I understand that the code does not contain dropout/batch normalization layers, model.eval() is still called for completeness.

for the code below:
```
epochs = 5
for t in range(epochs):
    print(f"Epoch {t+1}\n-------------------------------")
    train(train_dataloader, model, loss_fn, optimizer)
    test(test_dataloader, model, loss_fn)
print("Done!")
```

In the first epoch, train() is called, and then test() is called. And model.eval() is called within function test(), I understand that some layers are set to evaluation mode, it is necessary to call it before inference.

In the second epoch, the layers are still in evaluation mode, and train() is called. I think we should turn the evaluation mode off before training, but I don't see such code.

**Comment by guoyejun** [comment]
my point is that we need model.train() at the beginning of the function train()

**Comment by aditkumar72** [comment]
#1602

**Comment by holly1238** [comment]
Hey @suraj813, can you take a look and respond to this? Thanks!
